cmd/main.go

```diff
@@ -1,7 +1,9 @@
 package main
 
-import "fmt"
+import (
+	"news/internal/app"
```

Comment:
afaik "github.com" prefix is usually included into import prefix

Reply:
fixed